app/theme_defaults/_sub_fields.twig

```diff
@@ -30,15 +30,9 @@
 
 {# SECTION 2: MACROS #}
 
-{# If 'allowtwig' is true for this field, we'll need to parse it as
-   Twig here. Note, that we're doing this inside a macro, so the
-   snippet has a limited scope. No global variables, etc. #}
-{% macro parsetextfield(key, value, contenttype) %}
-    {% if contenttype.fields[key].allowtwig is defined and contenttype.fields[key].allowtwig == true %}
-        {% set value = include(template_from_string(value), sandboxed = true) %}
-    {% endif %}
-
-    <div {% if key is not empty %}data-bolt-field="{{key}}"{% endif %}>{{ value }}</div>
+{# Output a text-like field with the wrapper for live editing. #}
+{% macro textfield(object, key) %}
+    <div data-bolt-field="{{key}}">{{ object.getDecodedValue(key) }}</div>
```

Comment:
Twig style just needs `{{ object.decodedvalue(key) }}` 

… which TBH still doesn't sound like what it is … decoded from what to what, etc?